src/plugins/recordTypes/taxon/forms/default.jsx

```diff
@@ -0,0 +1,152 @@
+import { defineMessages } from 'react-intl';
+
+const template = (configContext) => {
+  const {
+    React,
+  } = configContext.lib;
+
+  const {
+    Col,
+    Cols,
+    Panel,
+    Row,
+  } = configContext.layoutComponents;
+
+  const {
+    Field,
+    InputTable,
+  } = configContext.recordComponents;
+
+  return (
+    <Field name="document">
+      <Panel name="info" collapsible>
+        <Field name="taxonTermGroupList">
+          <Field name="taxonTermGroup">
```

Comment:
I want to reorganize these fields, to look like ucjeps. Need fewer fields per row.